### PR TITLE
Integrate feedback from reviews and audits.

### PR DESCRIPTION
**Pull request by acote88**

### CODE REVIEW

File: contracts/TokenSaleConfig.sol [hunk 1]
```diff
@@ -38,4 +38,9 @@ contract TokenSaleConfig is SimpleTokenConfig {
     // For the public sale, tokens are priced at 0.0833 USD/token.
     // So if we have 300 USD/ETH -> 300,000 USD/KETH / 0.0833 USD/token = ~3,600,000
     uint256 public constant TOKENS_PER_KETHER         = 3600000;
+
+    // Constant used by buyTokens as part of the cost <-> tokens conversion.
+    // 18 for ETH -> WEI, TOKEN_DECIMALS (8 for Simple Token), 3 for the K in tokensPerKEther.
```
Comment (benjaminbollen) [inline contracts/TokenSaleConfig.sol]: correct to `18 for Simple Token`

**Review by benjaminbollen — COMMENTED** [review]
minor typo correction in comment

File: contracts/TokenSale.sol [hunk 2]
```diff
@@ -443,7 +447,7 @@ contract TokenSale is OpsManaged, Pausable, TokenSaleConfig { // Pausable is als
 
 
     // Allows the admin to finalize the sale and complete allocations.
-    // The owner will also need to finalize the token contract so that
+    // The admin will also need to finalize the token contract so that
```
Comment (jasonklein) [inline contracts/TokenSale.sol]: Suggest being clear that  the `admin` that needs to `finalize` the token contract is `SimpleToken.admin` and not necessarily `TokenSale.admin`

**Review by jasonklein — CHANGES_REQUESTED** [review]
Only one small comment

**Review by jasonklein — APPROVED** [review]

**Pull request merged**